wercker.yml

```diff
@@ -42,6 +44,17 @@ build:
         cwd: bootstrap/tests
         code: |
           /bin/bash test.sh
+    - capgemini/apollo-deploy:
+        cloud: aws-public
+        artifact_name: capgemini/apollo-ubuntu-14.04-amd64
+        artifact_version: 6
+        terraform_version: 0.5.3
```

Comment:
probably should bump this inline with the code we merged the other day in common.sh